netpoll/handle.go

```diff
@@ -16,20 +16,31 @@ type filer interface {
 type Desc struct {
 	file  *os.File
 	event Event
+	fd    int
 }
 
 // NewDesc creates descriptor from custom fd.
 func NewDesc(fd uintptr, ev Event) *Desc {
-	return &Desc{os.NewFile(fd, ""), ev}
+	return &Desc{file: os.NewFile(fd, ""), event: ev, fd: int(fd)}
+}
+
+// NewDescFile creates descriptor from os.File
+func NewDescFile(file *os.File, ev Event) *Desc {
```

Comment:
Is it really necessary to provide additional `Desc` constructor here? Why just don't setup `fd` field of the struct right in place where it is created and remove whole `Fd()` getter method?